test/runner/test.py

```diff
@@ -8,6 +8,8 @@
 import os
 import sys
 
+sys.path.append(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
I don't think this will have the desired effect. I believe @abadger is concerned that `bin/ansible-test` would become a real file instead of a symbolic link, in which case this would append the `bin` dir to the path.

Reply:
Feel free to make that change, because I don't see how you'd know exactly what is needed. Unless we try to match path names, but that seems the wrong thing to do too here.

Reply:
@abadger Can you clarify what it is you're looking for here?